core/chains/evm/types/block_json_bechmark_test_results.txt

```diff
@@ -0,0 +1,13 @@
+goos: darwin
+goarch: arm64
+pkg: github.com/smartcontractkit/chainlink/core/chains/evm/types
+BenchmarkBlock_Small_JSONUnmarshal
+BenchmarkBlock_Small_JSONUnmarshal-10               8024            145610 ns/op            2000 B/op         45 allocs/op
+BenchmarkBlock_Medium_JSONUnmarshal
+BenchmarkBlock_Medium_JSONUnmarshal-10               369           3209940 ns/op           52304 B/op        867 allocs/op
+BenchmarkBlock_Large_JSONUnmarshal
+BenchmarkBlock_Large_JSONUnmarshal-10                 45          25696396 ns/op          416970 B/op       6701 allocs/op
+BenchmarkBlock_XL_JSONUnmarshal
+BenchmarkBlock_XL_JSONUnmarshal-10                     5         207319750 ns/op         3177817 B/op      53303 allocs/op
+PASS
+ok      github.com/smartcontractkit/chainlink/core/chains/evm/types     7.437s
```

Comment:
Is there some reason this needs to be committed?

Reply:
Yeah I think it is usually most accurate to run these as close to one another as possible, and treat them as ephemeral. As long as this doesn't get left behind though :shrug: